### PR TITLE
Add init schema support for SQL Server in worker

### PR DESCRIPTION
**Pull request by alishakawaguchi**
Fast follows
- show init schema run context info on run page
- investigate & fix mssql truncate bug

### CODE REVIEW

File: backend/pkg/sqlmanager/mssql/generate-sql.go [hunk 1]
```diff
@@ -0,0 +1,383 @@
+package sqlmanager_mssql
```
Comment (nickzelei) [inline backend/pkg/sqlmanager/mssql/generate-sql.go:1]: here be dragons 🐉

File: backend/pkg/sqlmanager/shared/types.go [hunk 2]
```diff
@@ -166,6 +163,9 @@ func (s *SchemaTableDataTypeResponse) GetStatements() []string {
 	for _, domain := range s.Domains {
 		output = append(output, domain.Definition)
 	}
+	for _, fn := range s.Functions {
```
Comment (nickzelei) [inline backend/pkg/sqlmanager/shared/types.go:166]: Guessing this is to handle the case where functions are using things like enums, domains, etc?

Comment (alishakawaguchi) [reply backend/pkg/sqlmanager/shared/types.go:166]: exactly

**Review by nickzelei — APPROVED** [review]

File: worker/pkg/workflows/datasync/activities/run-sql-init-table-stmts/init-statement-builder.go [hunk 3]
```diff
@@ -379,9 +401,46 @@ func (b *initStatementBuilder) RunSqlInitTableStatements(
...
+	accountId string,
+) error {
+	bits, err := json.Marshal(initschemaRunContexts)
+	if err != nil {
+		return fmt.Errorf("failed to marshal init schema run context: %w", err)
+	}
+	_, err = b.jobclient.SetRunContext(ctx, connect.NewRequest(&mgmtv1alpha1.SetRunContextRequest{
+		Id: &mgmtv1alpha1.RunContextKey{
+			JobRunId:   b.workflowId,
+			ExternalId: "init-schema",
```
Comment (nickzelei) [inline worker/pkg/workflows/datasync/activities/run-sql-init-table-stmts/init-statement-builder.go:433]:
```suggestion
			ExternalId: "init-schema-report",
```

**Review by nickzelei — APPROVED** [review]

**Pull request merged**